Tools/LambdaTestTool-v2/tests/Amazon.Lambda.TestTool.IntegrationTests/ApiGatewayResponseExtensionsAdditionalTests.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using Amazon.Lambda.APIGatewayEvents;
+using Microsoft.AspNetCore.Http;
+using System.Text.Json;
+using Amazon.Lambda.TestTool.Extensions;
+using Amazon.Lambda.TestTool.Models;
+using System.Text;
+
+namespace Amazon.Lambda.TestTool.IntegrationTests
+{
+    [Collection("ApiGateway Integration Tests")]
+    public class ApiGatewayResponseExtensionsAdditionalTests
+    {
+        private readonly ApiGatewayIntegrationTestFixture _fixture;
+        private readonly HttpClient _httpClient;
+
+        public ApiGatewayResponseExtensionsAdditionalTests(ApiGatewayIntegrationTestFixture fixture)
+        {
+            _fixture = fixture;
+            _httpClient = new HttpClient();
+        }
+
+        //[Fact]
+        //public async Task V2_SetsContentTypeApplicationJsonWhenNoStatusProvided()
```

Comment:
i commented this out for now so it can be referenced for future when we create the lambda -> api gateway response layer